### PR TITLE
Return to fs-extra in onRunnerEnd method

### PR DESCRIPTION
**Pull request by HannaTarasevich**
To resolve https://github.com/webdriverio-community/wdio-cucumberjs-json-reporter/issues/163 issue.
(issue: Sometimes JSON files generated empty or not full JSON generated (5 and higher versions only))

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import fs from 'node:fs/promises'
+import fsExtra from 'fs-extra'
```
Comment (christian-bromann) [inline src/index.ts:1]:
```suggestion
import fs from 'fs-extra'
```

File: src/index.ts [hunk 2]
```diff
@@ -22,6 +22,7 @@ import type { CucumberJsJsonReporterInterface, AttachmentType, CucumberAttachmen
 import type { HookStatsExtended, SuiteStatsExtended, TestStatsExtended } from './types/wdio.js'
 
 const log = logger('wdio-cucumberjs-json-reporter')
+const { existsSync, outputJsonSync, readJsonSync } = fsExtra
```
Comment (christian-bromann) [inline src/index.ts:25]: Let's avoid having extra assignment and just use e.g. `fs.outputJsonSync`

**Review by christian-bromann — COMMENTED** [review]
Two comments, otherwise this looks good

File: src/index.ts (hunk 1 above)
Comment (HannaTarasevich) [reply src/index.ts]: Good point, updated

File: src/index.ts (hunk 2 above)
Comment (HannaTarasevich) [reply src/index.ts]: Thank you, updated

**Review by christian-bromann — APPROVED** [review]
LGTM 👍

**Pull request merged**